lib/twisted/plugins/aqd.py

```diff
@@ -232,7 +232,9 @@ def stop_coverage():
                                        "aq_notifyd")
                 if not os.path.exists(notifyd):
                     notifyd = notifyd + ".py"
-                args = [notifyd, "--config", config.baseconfig]
```

Comment:
We ensure this by modifying the shebang in the aq_notifyd and aqd to have exact version of python interpreter. This is done by Makefile when we build release. But i see your point. As the sys.executable would give the value of a python that aqd.py run on and we do not do different versions for the 2 processes - i see no problem. Btw, one always can set 'run_aqnotifyd' config to false and run/start the aq_notifyd as separate service. @gombasg  do you see any issues with merge this one in?

Reply:
@urbonegi the point, as you understood, is that if you want to use virtualenv, you cannot resolve the path at build time. As the goal is to ensure that you use the same interpreter for boht, I don't see how my modification will create a problem!
setting `run_aqnotifyd`to false is just adding the management burden when it is so convenient to have the broker taking the control of everything.
I hope @gombasg will agree!